tests/e2e/cucumber/features/smoke/notifications.oc10.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Notifications
+  As a user
+  I want to be notified
+  About new things that concern me
+
+  Background:
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    And "Admin" sets the default folder for received shares to "Shares"
+    And "Admin" disables share auto accepting
+
+  Scenario: User should be able to read and dismiss notifications
+    Given "Alice" creates the following folder in personal space using API
+      | name                   |
+      | folder_to_shared       |
+    And "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" shares the following resource using the sidebar panel
+      | resource          | recipient | type | role    | resourceType |
+      | folder_to_shared  | Brian     | user | editor  | folder       |
+    When "Brian" logs in
+    Then "Brian" should see the following notifications
+      | message                                        |
+      | "Alice Hansen" invited you to view "folder_to_shared" |
+    And "Brian" marks all notifications as read
+    Then "Brian" should see the following notifications
+      | text |
```

Comment:
Huh ?

Reply:
Brian should see no notifications after clearing them, hence the empty list.

Reply:
Got you, Tests looks good to me. 
It's on you if you want to add something like 'should see no new notifications' as it would be easier to read. But that's totally on ya. Will approve anyways.

Reply:
Would prefer such an explicit step as well

Reply:
Alrighty, that's 2 against 1 😄